src/utils/DelegatedAccessRegistry.ts

```diff
@@ -0,0 +1,16 @@
+import { Contract, providers } from 'ethers'
+import * as DelegatedAccessRegistry from '../contracts/DelegatedAccessRegistry.sol/DelegatedAccessRegistry.json'
+import { Provider } from '@web3-react/types'
+
+const DelegatedAccessRegistryAbi = DelegatedAccessRegistry.abi
+const DelegatedAccessRegistryAddress = '0xB3042ecFC4Ba4ef213A38B1C2541E9234a6189cc'
+
+const getDelegatedAccessRegistryAt = (rawProvider: Provider): Contract => {
```

Comment:
`At`? `getDelegatedAccessRegistry` sounds more natural when the only arg is a provider. "At" is probably there because in the past (?) there was an address there, maybe?